packages/terra-responsive-element/src/terra-dev-site/doc/ResponsiveElement.doc.jsx

```diff
@@ -1,72 +1,30 @@
+/* eslint-disable import/no-extraneous-dependencies, import/no-webpack-loader-syntax, import/first, import/no-unresolved, import/extensions, import/no-duplicates */
```

Comment:
I prefer the ignores to just wrap the responsive table example and example src files. It doesn't look as pretty but is more targeted to which items actually need to be ignored.